README.md

```diff
@@ -45,49 +45,97 @@ The backend consists of the MySQL database and the Spring Boot app. You will als
 * There are two ways to create the MySQL database backend: **Use MySQL in a container in your cluster** *OR* **Use Bluemix MySQL**
 
 ## 1.1 Use MySQL in container
-**NOTE:** Leave the environment variables blank in the `compute-interest-api.yaml` and `account-summary.yaml`
 ```bash
 $ kubectl create -f account-database.yaml
 service "account-database" created
 deployment "account-database" created
 ```
+Default credentials are already encoded in base64 in secrets.yaml.
```

Comment:
We should probably put a disclaimer in there about best practices when managing sensitive information. 1) Don't store in github 2) base64 does not hide your secrets 3) Use password manager and kube secrets which separate your coding flow from your secrets management